test/data/test_feature_store.py

```diff
@@ -22,21 +26,23 @@ def __init__(self):
         self._tensor_attr_cls = MyTensorAttrNoGroupName
 
 
-def test_feature_store():
+@pytest.mark.parametrize('device', DEVICES)
+def test_feature_store(device):
```

Comment:
Do we need to parameterize on device here (in this file and in the graph store test file as well)? Seems like we are not really changing what we test based on the device.

Reply:
valid point, will adjust